pyspectrafuse/cluster_parquet_combine/combine_cluster_and_parquet.py

```
@@ -45,17 +51,23 @@ def read_cluster_tsv(tsv_file_path: str):
         clu_df.dropna(axis=0, inplace=True)  # 删除空行
         return clu_df
 
-    def inject_cluster_info(self, path_parquet, path_sdrf,  path_cluster_tsv, spectra_num=1000000, batch_size=200000):
+    def inject_cluster_info(self, path_parquet, clu_map_dict, path_sdrf, spectra_num=1000000, batch_size=200000):
         # read parquet file and the cluster result tsv file of Maracluster program.
+        print("inject cluster_info: path_parquet:", path_parquet)
+        if type(path_parquet) == list:
```

Ruff (0.14.10)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)